app.json

```diff
@@ -39,7 +40,12 @@
         "READ_EXTERNAL_STORAGE",
         "WRITE_EXTERNAL_STORAGE"
       ],
-      "jsEngine": "hermes"
+      "jsEngine": "hermes",
+      "config": {
+        "googleMaps": {
+          "apiKey": "abcdef"
```

Comment:
this needs to be changed individually in releases with a secret google maps api key